roles/openshift_health_checker/openshift_checks/etcd_traffic.py

```diff
@@ -0,0 +1,35 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class EtcdTraffic(OpenShiftCheck):
+    """Check if host is being affected by an increase in etcd traffic."""
+
+    name = "etcd_traffic"
+    tags = ["health", "etcd"]
+
+    @classmethod
+    def is_active(cls, task_vars):
+        """Skip hosts that do not have etcd or masters in their group names."""
+        group_names = get_var(task_vars, "group_names", default=[])
+        active = "masters" in group_names or "etcd" in group_names
+        return super(EtcdTraffic, cls).is_active(task_vars) and active
+
+    def run(self, tmp, task_vars):
+        match = self.module_executor("etcdlogs", {
+            "log_matcher": {
+                "regexp": "etcd: sync duration of \d+\.\d+s, expected less than 1s",  # pylint: disable=anomalous-backslash-in-string
```

Comment:
use `r'` so the backslashes aren't misinterpreted - or does that not work across the module invocation? i think it should be fine.